### PR TITLE
Correct/Update operations.md

### PR DESCRIPTION
**Pull request by dupuchba**
I think this was a mistake, any feedback welcomed :-)

### CODE REVIEW

File: Resources/doc/operations.md [hunk 1]
```diff
@@ -21,8 +21,7 @@ By default, the following operations are automatically enabled:
 ## Disabling operations
 
 If you want to disable some operations (e.g. the `DELETE` operation), you must register manually applicable operations using
-the operation factory class, `Dunglas\ApiBundle\Resource::addCollectionOperation()` and `Dunglas\ApiBundle\Resource::addCollectionOperation()`
-methods.
```
Comment (dunglas) [inline Resources/doc/operations.md]: Was `addItemOperation` IMO

Comment (dupuchba) [inline Resources/doc/operations.md]: `addItemOperation ` doesn't exist in my project, besides, doc speaks about the factory creating operations. Maybe I am lost ^_^

Comment (ogizanagi) [inline Resources/doc/operations.md]: It's `createItemOperation` and `createCollectionOperation`: https://github.com/dunglas/DunglasApiBundle/blob/master/Api/Operation/OperationFactory.php#L45-L77

Comment (dupuchba) [inline Resources/doc/operations.md]: yep :)

Comment (dupuchba) [inline Resources/doc/operations.md]: IMO It makes more sense the factory way

Comment (dunglas) [inline Resources/doc/operations.md]: In fact you must create an operation with the factory (`create*`) then register it in the resource (`init*`)

**Comment by dunglas** [comment]
Thank you @dupuchba. Can you squash your commits?

**Comment by dupuchba** [comment]
@dunglas done

**Comment by dunglas** [comment]
Thank you.

**Pull request merged**